src/meta/ClusterManager.cpp

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+
+#include "base/Base.h"
+#include "fs/FileUtils.h"
+#include "ClusterManager.h"
+
+namespace nebula {
+namespace meta {
+
+bool ClusterManager::loadOrCreateCluId() {
+    if (loadClusterId()) {
+        return true;
+    }
+    createClusterId();
+    return dumpClusterId();
+}
+
+
+bool ClusterManager::loadClusterId() {
+    int fd = ::open(clusterIdPath_.c_str(), O_RDONLY);
+    if (fd < 0) {
+        LOG(WARNING) << "loadClusterId failed!";
+        return false;
+    }
+    char buff[32] = {};
+    int len = ::read(fd, buff, sizeof(buff));
+    buff[len] = '\0';
```

Comment:
read will return a value less 0 if there is some error. maybe need check it.

Reply:
Yes, it's a problem! I have fixed it!